draft-ietf-acme-acme.md

```diff
@@ -1939,7 +1943,7 @@ own heuristics to decide which is optimal.
 ~~~~~~~~~~
 GET /acme/cert/asdf HTTP/1.1
 Host: example.com
-Accept: application/pkix-cert
+Accept: application/pem-certificate-chain
```

Comment:
Couldn't we list both?

Reply:
We could, but I think that would just add ambiguity to the example.